static/src/stylesheets/pasteup/palette/_src.scss

```diff
@@ -1,71 +1,74 @@
-$neutral-3:              #bdbdbd;
-$neutral-4:              #dcdcdc;
-$neutral-5:              #dfdfdf;
-$neutral-6:              #eaeaea;
-$neutral-7:              #f1f1f1;
-$neutral-8:              #f6f6f6;
+$neutral-1: #333333;
+$neutral-2: #767676;
+$neutral-3: #bdbdbd;
+$neutral-4: #dcdcdc;
+$neutral-5: #dfdfdf;
+$neutral-6: #eaeaea;
+$neutral-7: #f1f1f1;
+$neutral-8: #f6f6f6;
 
 // News palette
-$news-main-1:            #005689;
-$news-main-2:            #4bc6df;
-$news-support-1:         #aad8f1;
-$news-support-2:         #197caa;
-$news-support-3:         #69d1ca;
-$news-support-4:         #66a998;
-$news-support-5:         #aad801;
-$news-support-6:         #63717a;
-$news-support-7:         #484f53;
+$news-main-1: #005689;
+$news-main-2: #4bc6df;
+$news-support-1: #aad8f1;
+$news-support-2: #197caa;
+$news-support-3: #69d1ca;
+$news-support-4: #66a998;
+$news-support-5: #aad801;
+$news-support-6: #63717a;
+$news-support-7: #484f53;
 
 // Features palette
-$features-main-1:        #7d0068;
-$features-main-2:        #b82266;
-$features-support-1:     #951c55;
-$features-support-2:     #4e0375;
-$features-support-3:     #fdadba;
-$features-support-4:     #dc2a7d;
+$features-main-1: #7d0068;
+$features-main-2: #b82266;
+$features-support-1: #951c55;
+$features-support-2: #4e0375;
+$features-support-3: #fdadba;
+$features-support-4: #dc2a7d;
 
 // Comment palette
-$comment-main-1:         #c05303;
-$comment-main-2:         #ff9b0b;
-$comment-support-1:      #7d7569;
-$comment-support-2:      #efefec;
-$comment-support-3:      #ffce4b;
-$comment-support-4:      #e6711b;
+$comment-main-1: #c05303;
+$comment-main-2: #ff9b0b;
+$comment-support-1: #7d7569;
+$comment-support-2: #efefec;
+$comment-support-3: #ffce4b;
+$comment-support-4: #e6711b;
 
 // Multimedia palette
-$multimedia-main-1:      #333333;
-$multimedia-main-2:      #ffbb00;
-$multimedia-support-1:   #c5d4ea;
-$multimedia-support-2:   #507892;
-$multimedia-support-3:   #002c59;
-$multimedia-support-4:   #484848;
-$multimedia-support-5:   #161616;
+$multimedia-main-1: #333333;
+$multimedia-main-2: #ffbb00;
+$multimedia-support-1: #c5d4ea;
+$multimedia-support-2: #507892;
+$multimedia-support-3: #002c59;
+$multimedia-support-4: #484848;
+$multimedia-support-5: #161616;
 
 // Live palette
-$live-main-1:            #b51800;
-$live-main-2:            #cc2b12;
-$live-support-1:         #ff5b3a;
-$live-support-2:         #800c0c;
-$live-support-3:         #a60947;
+$live-main-1: #b51800;
+$live-main-2: #cc2b12;
+$live-support-1: #ff5b3a;
+$live-support-2: #800c0c;
+$live-support-3: #a60947;
 
 // Maps palette
-$maps-main-1:            #1c6326;
-$maps-main-2:            #298422;
-$maps-support-1:         #ceb41d;
-$maps-support-2:         #a9af2b;
-$maps-support-3:         #5ebfba;
-$maps-support-4:         #72af7e;
+$maps-main-1: #1c6326;
+$maps-main-2: #298422;
+$maps-support-1: #ceb41d;
+$maps-support-2: #a9af2b;
+$maps-support-3: #5ebfba;
+$maps-support-4: #72af7e;
```

Comment:
Am I the only one who will be sad to lose the pretty alignment of hex values here?

Reply:
Ah didn't notice that, will fix

Reply:
agree – wasn't sure whether to mention it. as a rule, it's ideal if PRs can be limited to the changes in functionality (rather than white space). it's make it much easier to see what the PR is actually doing. always a worth a look at the changes in the github diff to make sure you've not accidentally added unintended changes to you PR